cmd/main.go

```diff
@@ -85,9 +85,18 @@ func main() {
 		BaseURL:    nil,
 	}
 
-	wg := sync.WaitGroup{}
-	wg.Add(1)
 	metricServer(metricsAddr)
+	wg := sync.WaitGroup{}
+	c := make(chan os.Signal)
```

Comment:
add a buffer to fix lint error.
`c := make(chan os.Signal, 1)`

Reply:
fixed